GcmEndpoints/README.md

```diff
@@ -263,7 +131,7 @@ sign-in with your Google Account. Choose an account and sign in.<br>
 
 ## 2.6. Testing against a deployed backend
 
-Once you have deployed your backend to App Engine, you can connect your Android app to it by modifying `GcmRegistrationAsyncTask` class defined in section 2 above. In particular, replace the lines
+Once you have deployed your backend to App Engine, you can connect your Android app to it by modifying `RegistrationIntentService` class explained in section 2 above. In particular, replace the lines
 ```java
 Registration.Builder builder = new Registration.Builder(AndroidHttp.newCompatibleTransport(), new AndroidJsonFactory(), null)
```

Comment:
I just realized there's some inconsistency is the code snipped show here vs what's above, can we sync that up in this CL?